package.json

```diff
@@ -15,28 +15,28 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "@xmldom/xmldom": "^0.8.1",
-    "async": "^3.2.3",
+    "@xmldom/xmldom": "^0.8.6",
+    "async": "^3.2.4",
     "colors": "1.4.0",
     "fs-extra": "~9.1.0",
     "request": "~2.88.0",
-    "semver": "~7.3.5",
+    "semver": "~7.3.8",
     "sprintf": "^0.1.5",
     "temp": "~0.9.4",
-    "uuid": "~8.3.2",
+    "uuid": "~9.0.0",
```

Comment:
I'd have to test this, but I'm hesitant to blind merge a major bump.

Reply:
only tested it on Linux. btw: I don't see where it is used